Index: tuya_sdk-1.0.0/CMakeLists.txt
===================================================================
--- tuya_sdk-1.0.0.orig/CMakeLists.txt
+++ tuya_sdk-1.0.0/CMakeLists.txt
@@ -19,9 +19,11 @@ add_compile_options( -g )
 # set(CMAKE_CXX_COMPILER arm-linux-androideabi-g++)
 
 # Do not allow in-source build.
-if( ${PROJECT_SOURCE_DIR} STREQUAL ${PROJECT_BINARY_DIR} )
-    message( FATAL_ERROR "In-source build is not allowed. Please build in a separate directory, such as ${PROJECT_SOURCE_DIR}/build." )
-endif()
+#if( ${PROJECT_SOURCE_DIR} STREQUAL ${PROJECT_BINARY_DIR} )
+#    message( FATAL_ERROR "In-source build is not allowed. Please build in a separate directory, such as ${PROJECT_SOURCE_DIR}/build." )
+#endif()
+
+install(CODE "MESSAGE(\"Creted empty install target.\")")
 
 # Set global path variables.
 get_filename_component(__root_dir "${CMAKE_CURRENT_LIST_DIR}" ABSOLUTE)
